.github/workflows/deploy.yml

```diff
@@ -42,18 +42,17 @@ jobs:
         git tag --list | tail -n 20
         echo ==========
         buildscripts/ci/release/make_tag_name.sh | tee -a "${GITHUB_OUTPUT}"
-        echo ==========
         echo "${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/commit/${GITHUB_SHA}"
         echo ==========
         if [[ "${CREATE_TAG}" == 'true' ]]; then
           echo "Tag will be created."
         else
-          echo "Tag will not be created."
+          echo "Warning: Tag will NOT be created."
```

Comment:
Maybe this would be nice to use: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-warning-message